StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1648UnitTests.cs

```diff
@@ -90,7 +215,6 @@ public async Task TestTypeWithEmptyBaseListAndCrefAttributeAsync(string declarat
         }
 
         [Theory]
-        [InlineData("Test() { }")]
```

Comment:
📝 Let's add this back (so we have coverage of constructors), but change the signature (so it doesn't have a matching base type constructor)

For example:

```cs
[InlineData("Test(int ignored) { }")]
```